packages/lambdas/src/inbound-patient-discovery.ts

```diff
@@ -17,6 +17,7 @@ export const handler = Sentry.AWSLambda.wrapHandler(async (event: APIGatewayProx
     id: payload.id,
     timestamp: payload.timestamp,
     samlAttributes: payload.samlAttributes,
+    patientResource: payload.patientResource,
```

Comment:
what does this have to do with mTLS?

Reply:
ah nothing was just an improvement to the PD logic. Was a lingering change I just added herer